.github/workflows/backend-ci-cd-prod.yml

```
@@ -69,25 +78,22 @@ jobs:
-          container_id=$(docker ps --filter "publish=80" --format "{{.ID}}")
-          if [ -n "$container_id" ]; then
-            echo "컨테이너 중지: $container_id"
-            docker stop $container_id
-          else
-            echo "중지할 컨테이너 없음 (포트 80)"
-          fi
-
-      - name: Run docker image
+          echo "prod 컨테이너 중지 중..."
+          docker stop prod && echo "prod 컨테이너 중지 완료" || echo "prod 컨테이너 없음"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)